### PR TITLE
Changed all label formats to match the original.

### PR DESCRIPTION
**Pull request by DavidJSolano**
This PR fixes issue #655. The new labels look like this: 
Screenshot_20221120_084817
Screenshot_20221120_084831

### CODE REVIEW

**Review by EthanThatOneKid — APPROVED** [review]
Nice!! Hope you had an epic Thanksgiving break 😎

File: src/lib/components/blog/labels.svelte [hunk 1]
```diff
@@ -34,15 +35,18 @@
     font-size: 1em;
 
     a {
-      background-color: var(--acm-light);
-      border-radius: 5px;
-      padding: 0.3em;
+      background-color: #eeeeee;
```
Comment (jaasonw) [inline src/lib/components/blog/labels.svelte]: suggest going back to acm-light or using acm-canvas here, avoid having "magic numbers" or mix of different off white throughout the codebase whenever possible

Comment (jaasonw) [reply src/lib/components/blog/labels.svelte]: also makes it dark mode reactive

File: src/lib/components/blog/labels.svelte [hunk 2]
```diff
@@ -34,15 +35,18 @@
...
+      color: #212121;
```
Comment (jaasonw) [inline src/lib/components/blog/labels.svelte]: remove this line, for same reasons as above

File: src/lib/components/blog/labels.svelte [hunk 3]
```diff
@@ -34,15 +35,18 @@
...
+      border-radius: var(--size-sm);
+      border: 2px solid #e0e0e0;
```
Comment (jaasonw) [inline src/lib/components/blog/labels.svelte:41]: same reasons as above, maybe experiment if mixing acm-light and acm-canvas would produce a similar effect?
/

File: src/lib/components/blog/labels.svelte [hunk 4]
```diff
@@ -34,15 +35,18 @@
...
+      padding: 0.25em 0.8em 0.25em 0.8em;
       text-decoration: none;
     }
 
     .selected {
-      background-color: var(--acm-gray);
-      color: var(--perma-light);
+      background-color: #81d4fa;
+      border-color: #4fc3f7;
+      color: #212121;
```
Comment (jaasonw) [inline src/lib/components/blog/labels.svelte]: var(--perma-dark) here instead

File: src/lib/components/blog/labels.svelte [hunk 5]
```diff
@@ -34,15 +35,18 @@
...
+      border-radius: var(--size-sm);
+      border: 2px solid #e0e0e0;
+      padding: 0.25em 0.8em 0.25em 0.8em;
       text-decoration: none;
     }
 
     .selected {
-      background-color: var(--acm-gray);
-      color: var(--perma-light);
+      background-color: #81d4fa;
```
Comment (jaasonw) [inline src/lib/components/blog/labels.svelte:47]: how does using var(--acm-blue) and var(--acm-bluer) instead here look?

**Review by jaasonw — CHANGES_REQUESTED** [review]

**Comment by DavidJSolano** [comment]
Screenshot_20221206_123739
Screenshot_20221206_123754
This is how the new labels look.

File: src/lib/components/blog/labels.svelte (hunk 5 above)
Comment (DavidJSolano) [reply src/lib/components/blog/labels.svelte:47]: All the other changes look pretty good, not sure if I'm rocking with this change though.
Screenshot_20221206_011004

File: src/lib/components/blog/labels.svelte (hunk 1 above)
Comment (DavidJSolano) [reply src/lib/components/blog/labels.svelte]: Got it.

File: src/lib/components/blog/labels.svelte (hunk 2 above)
Comment (DavidJSolano) [reply src/lib/components/blog/labels.svelte]: Done.

File: src/lib/components/blog/labels.svelte (hunk 3 above)
Comment (DavidJSolano) [reply src/lib/components/blog/labels.svelte:41]: You got it.

**Review by EthanThatOneKid — APPROVED** [review]
> Screenshot_20221206_123739 Screenshot_20221206_123754 This is how the new labels look.

Nice! Thanks for also making sure to support dark mode :)

**Review by jaasonw — CHANGES_REQUESTED** [review]
sorry, i shoulda said this earlier but can you reflect the same changes in labelfield.svelte, or maybe split the label into its own component to avoid duplication?

image

**Comment by DavidJSolano** [comment]
Alright, I duplicated these changes to the labels in labelfield.svelte. I think it's fair to mention that the selected labels have an altered look when changed to strictly ACM colors, not sure how you guys feel about this look. Should I revert it back to the original colors?
Screenshot_20221206_011004
Let me know.

**Comment by EthanThatOneKid** [comment]
> Should I revert it back to the original colors? 

Thanks for following up, @DavidJSolano! I think we should go back to the original colors.

**Comment by DavidJSolano** [comment]
Okay, this is what the final product looks like after reverting the selected labels to their original colors and making them compatible with light/dark mode.
Screenshot_20221211_063807
Screenshot_20221211_063752

File: src/routes/(site)/blog/labelfield.svelte [hunk 6]
```diff
@@ -107,6 +107,7 @@
           transition: 1s ease-in-out;
           background-color: #81d4fa;
           border-color: #4fc3f7;
+          color: var(--acm-dark)
```
Comment (jaasonw) [inline src/routes/(site)/blog/labelfield.svelte]: looks like you're missing a semicolon here

**Review by jaasonw — CHANGES_REQUESTED** [review]

File: src/routes/(site)/blog/labelfield.svelte (hunk 6 above)
Comment (DavidJSolano) [reply src/routes/(site)/blog/labelfield.svelte]: Oops, thanks for pointing that out.

**Review by jaasonw — APPROVED** [review]
sexy

**Pull request merged**